proxy.php

```diff
@@ -119,7 +123,21 @@
     }
 
     echo $content;
+}
+
+function sanitizeContent($content)
+{
+    global $TOKEN_AUTH;
+    global $PIWIK_URL;
+    global $PROXY_URL;
+
+    $matomoHost = parse_url($PIWIK_URL, PHP_URL_HOST);
+    $proxyHost = parse_url($PROXY_URL, PHP_URL_HOST);
 
+    $content = str_replace($TOKEN_AUTH, '<token>', $content);
```

Comment:
would this mess up with a set `content-length`? I wonder if we need to use something like `str_pad()` to keep the same length or adjust the content-length header?

Reply:
Yeah, would mess it up, will make the content-length dynamic.